pages/agent/v3/elastic_ci_aws.md.erb

```diff
@@ -0,0 +1,260 @@
+# Elastic CI Stack for AWS
+
+The Buildkite Elastic CI Stack gives you a private, autoscaling [Buildkite Agent](/docs/agent/v3) cluster. Use it to parallelize legacy tests across hundreds of nodes, run tests and deployments for all your Linux-based services and apps, or run AWS ops tasks.
+
+{:toc}
+
+
+## Quickstart
+
+See the [Elastic CI Stack for AWS tutorial](/docs/tutorials/elastic_ci_stack_aws) for a step-by-step guide, or jump straight in:
+
+
+<!-- vale off -->
+<!-- alex ignore master -->
+
+<a href="https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=buildkite&templateURL=https://s3.amazonaws.com/buildkite-aws-stack/latest/aws-stack.yml"><%= image "launch-stack.svg", alt: "Launch stack button" %></a>
+
+<!-- vale on -->    
+
+> The stack creates its own VPC (virtual private cloud) by default. Best practice involves setting up a separate development AWS account and using role switching and consolidated billing. Refer to the [Delegate Access Across AWS Accounts tutorial](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html) for more information.
+
+
+If you want to use the [AWS CLI](https://aws.amazon.com/cli/), download [`config.json.example`](https://github.com/buildkite/elastic-ci-stack-for-aws/blob/master/config.json.example), rename it to `config.json`, add your values, and then run the below command:
```

Comment:
I'm still slight caught up on this. Should it be "use AWS CLI instead, " ...

And "add your values" is vague. You need to add the Agent Token, are the others required?